### ISSUE TITLE
Specifying No cache strategy during vpool creation still forces 1GB as Read Cache and allocates a file of 1GB

### ISSUE THREAD

**Issue by pploegaert** [openvstorage/framework#683]
vpool creation wizard summary:

```
Validate the basic vPool information.

Name: vp1
Backend Type: Open vStorage Backend
Backend: alba
Fragment cache on read: Inactive
Fragment cache on write: Inactive
Preset: default
Validate whether the correct Storage Router was selected.

Storage Router: ovs54c
Validate whether the required cache sizes and connection information are correct.

Read Cache: 1 GiB
Write Cache: 1 GiB
Storage IP: 172.20.54.250
Validate the configuration settings.

Cache Strategy: No Cache
Dedupe Mode: Non-Deduped
SCO Size: 4 MiB
Cluster Size: 4 KiB
Write Buffer: 128 MiB
DTL: Disabled
```

After creation read cache file is still allocated while No cache strategy was requested

```
root@ovs54c:/mnt# du -sh ssd?/*
2.4M    ssd1/arakoon
16K ssd1/lost+found
4.0K    ssd1/vp1_db_md_1
16K ssd1/vp1_db_mds_1
4.0K    ssd1/vp1_db_tlog_1
1001M   ssd1/vp1_read_1
4.0K    ssd1/vp1_scrub_1
16K ssd2/lost+found
4.0K    ssd2/vp1_write_dtl_1
4.0K    ssd2/vp1_write_fcache_1
4.0K    ssd2/vp1_write_fd_1
12K ssd2/vp1_write_sco_1
```

**Comment by wimpers** [comment]
- change the wizard so no read cache has to be specified if no cache was selected in the previous step
- on vdisk level don't allow to overrule

**Comment by khenderick** [comment]
Resolved by #729

**Comment by deepshirsath** [comment]
**PASSED** on **_Open vStorage: 2.7.1-fargo.2-1_**
- vPool creation wizard summary:

![frag](https://cloud.githubusercontent.com/assets/14201912/17620507/6edd53c2-60aa-11e6-881d-6105b321bebb.PNG)

![no-cache](https://cloud.githubusercontent.com/assets/14201912/17620422/e28d20f0-60a9-11e6-9fff-611a90364893.PNG)

![no-cache-1](https://cloud.githubusercontent.com/assets/14201912/17620432/e9d2ea2a-60a9-11e6-8427-d402ffbfdebd.PNG)
- After vPool creation read cache file not located while no cache strategy was selected.

![du-sh](https://cloud.githubusercontent.com/assets/14201912/17620468/2330eefc-60aa-11e6-90e9-e27002c57c20.PNG)
